pkg/cliplugins/workload/cmd/cmd.go

```diff
@@ -32,6 +32,18 @@ var (
 	syncExample = `
 	# Ensure a syncer is running on the specified workload cluster.
 	%[1]s workload sync <workload-cluster-name> --syncer-image <kcp-syncer-image>
+`
+	cordonExample = `
+	# Mark a workload cluster as unschedulable.
+	%[1]s workload cordon <workload-cluster-name>
+`
+	uncordonExample = `
+	# Mark a workload cluster as schedulable.
+	%[1]s workload uncordon <workload-cluster-name>
+`
+	drainExample = `
+	# Drain a workload cluster in preparation for maintenance.
```

Comment:
more like "Start draining". This command does not wait. We might want to add waiting in the future vi a `--wait`

Reply:
+1